src/components/stepforms/widgets/DateComponents/RelativeDateRangeForm.vue

```diff
@@ -99,21 +104,24 @@ export default class RelativeDateRangeForm extends Vue {
     this.$emit('input', { ...this.value, date: value });
   }
 
-  get directions() {
-    return [
-      { label: 'until', value: -1 },
-      { label: 'from', value: +1 },
-    ];
+  get operators() {
+    return [RELATIVE_DATE_RANGE_OPERATORS.until, RELATIVE_DATE_RANGE_OPERATORS.from];
   }
 
-  get rangeDirection() {
-    return this.value.quantity >= 0 ? this.directions[1] : this.directions[0];
+  get operator() {
+    // Current value should have an operator but we introduced it without at first,
+    // relying entierly on quantity's sign to encode it.
+    // We keep this fallback as a reminder of this mistake until someone decide that comments
+    // is not the place for keeping track of our stuttering torward Clean Code :tm:
+    const fallbackOperator = this.value.quantity >= 0 ? this.operators[1] : this.operators[0];
```

Comment:
Detail: Using directly `RELATIVE_DATE_RANGE_OPERATORS.until` and `RELATIVE_DATE_RANGE_OPERATORS.from` could be a bit more readable (avoid relying on the order of operators and need to check this order above)